examples/express-api-reference/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS builder
+WORKDIR /app
+
+COPY pnpm-lock.yaml .
+COPY pnpm-workspace.yaml .
+COPY package.json .
+COPY tsconfig.json .
+COPY packages/api-reference ./packages/api-reference
+COPY packages/components ./packages/components
+COPY packages/themes ./packages/themes
+COPY packages/oas-utils ./packages/oas-utils
+COPY packages/build-tooling ./packages/build-tooling
+COPY packages/express-api-reference ./packages/express-api-reference
+COPY examples/express-api-reference ./examples/express-api-reference
+
+RUN --mount=type=cache,id=pnpm,target=~/.pnpm-store \
+    pnpm --filter @scalar-examples/express-api-reference install --frozen-lockfile && \
+    pnpm --filter @scalar-examples/express-api-reference build
+
+
+FROM node:18-bullseye-slim AS runner
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+
+RUN npm install -g pnpm
+ENV NODE_ENV production
+
+WORKDIR /app
+
+# Copy root node modules and any utilized packages
+COPY --from=builder --chown=node:node /app/node_modules /app/node_modules
```

Comment:
Can we just chown the /app directory here? These copy commands are very verbose.

Reply:
Are you suggesting copy all the files and then `chown` the directory? Setting the owner during the copy seems to be best practice although slightly verbose.

Reply:
You should be able to chown the directory then copy the files

Reply:
Or do it after. I don't see any benefit to this approach from a security standpoint or otherwise. Just very verbose.